packages/react/src/utils.ts

```diff
@@ -422,3 +422,31 @@ export const getModelManager = (
 ): AnyModelManager | undefined => {
   return get(apiClient, namespaceDataPath([apiIdentifier], namespace).join("."));
 };
+
+export class Logger {
```

Comment:
@infiton You cool with this? I figured this is the most elegant way to handle enabling and disabling the debug for useActionForm

Reply:
You don't want to just check the flag each time? And use console directly? I doubt this increase bundle size much at all, but even then not sure an extra class really buys us much